packages/common/src/Class.ts

```diff
@@ -64,7 +64,7 @@ export class Class {
     }
 
     cssTag(id: string): string {
-        return id.replace(/[^a-z0-9]/g, (s) => {
+        return ("" + id).replace(/[^a-z0-9]/g, (s) => {
```

Comment:
Why was this necessary?

Reply:
I think its for the case where id is a number.